### PR TITLE
Test update agentid

### PR DESCRIPTION
**Pull request by betsyknoke**
Update Android tests to run tests individually, to handle autostart, and to accommodate running on the test lab.

### CODE REVIEW

**Review by mmorehou — APPROVED** [review]
Nice work!

File: androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt [hunk 1]
```diff
@@ -155,22 +165,13 @@ class ComposePrototypeUITest : BaseTests() {
...
-        opening {
-            verifyPageIsVisible()
-            enterDeploymentID()
-            selectView(testBedViewText)
-        }
-        messenger {
-            verifyPageIsVisible()
-        }
+    // Adjusting the test name to force this test to run first
+    fun testAAVerifyAutoStart() {
```
Comment (AfanasievAnton) [inline androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt:173]: is it a typo with double `A` here?

**Review by AfanasievAnton — APPROVED** [review]

File: androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt (hunk 1 above)
Comment (betsyknoke) [reply androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt:173]: I adjusted the name of the test to force it to run first. (tests running in alphabetic order)

Comment (AfanasievAnton) [reply androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt:173]: Are digits run before letters? We can make it more clear by naming it `test1VerifyAutostart`, but it is not critical for me.

**Pull request merged**